src/content/docs/orgs.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: Organizations
+description: Learn how to manage organizations in Shorebird
+---
+
+Shorebird's Pro tier unlocks new features for collaborating with your peers
+including:
+
+- Create unlimited organizations
+- Add unlimited members to your organizations
+- Manage roles for your organization members
+- Add unlimited collaborators to an individual app
+- Manage roles for your app collaborators
```

Comment:
I'm not sure what the proper grammatical term is for this, but I feel like "collaborating" and "create" should have the same verb form (either collaborate+create or collaborating+creating)